### ISSUE TITLE
silk profile is not work, with dango-version 2.0.2 and django-silk version 2.0.0

### ISSUE THREAD

**Issue by laiyijie** [jazzband/django-silk#277]
![image](https://user-images.githubusercontent.com/7499201/39040337-ab64d4a6-44b9-11e8-9161-5f94c9733fd7.png)
when I using the decorator in the view function like this:
![image](https://user-images.githubusercontent.com/7499201/39040415-cc592d38-44b9-11e8-9f9f-a368b76ca94d.png)

and I also tried to set 
SILKY_PYTHON_PROFILER = True
SILKY_PYTHON_PROFILER_BINARY = True
and the profile is not work at all
"No Silk profiling was performed for this request. Use the silk_profile decorator/context manager to do so."

**Comment by l-vincent-l** [comment]
I have the same issue, and no idea how to solve it.
Any help will be appreciated.

**Comment by danielbradburn** [comment]
I am unable to replicate this issue, for example the test ``project`` in the silk repo works correctly under django 2.0.2

Perhaps you could try adding this to your ``settings.py`` to ensure the profiles can be correctly saved?

```
    SILKY_PYTHON_PROFILER_RESULT_PATH = os.path.join(BASE_DIR, '.profiles')
    with contextlib.suppress(OSError):
        os.makedirs(SILKY_PYTHON_PROFILER_RESULT_PATH)
```

**Comment by sur0g** [comment]
For Django==2.0.2 and django-silk==2.0.0.
I've done some research with this issue and found some error in lib\site-packages\silk\collector.py. It uses direct assignment to one of the class attributes though it's forbidden in ***\Lib\site-packages\django\db\models\fields\related_descriptors.py directly:
```python
def __set__(self, instance, value):
	raise TypeError(
		'Direct assignment to the %s is prohibited. Use %s.set() instead.'
		% self._get_set_deprecation_msg_params(),
	)
```
It helps if I change the row 183 of lib\site-packages\silk\collector.py to 
```python
profile.queries.set(profile_query_models)
```

**Comment by albertyw** [comment]
That was fixed in #275.  I think it's time to make a new release.  @avelis ?

**Comment by avelis** [comment]
@albertyw Sounds good to me.
